lib/base-compiler.js

```diff
@@ -421,7 +421,10 @@ class BaseCompiler {
     }
 
     isCfgCompiler(compilerVersion) {
-        return compilerVersion.includes("clang") || compilerVersion.indexOf("g++") === 0;
+        return compilerVersion.includes("clang") ||
+               compilerVersion.indexOf("g++") === 0 ||
+               compilerVersion.indexOf("gdc") === 0 ||
+               compilerVersion.indexOf("LDC") === 0;
```

Comment:
I think you can remove the LDC check here, and we could use both PRs indeed. LDC would just return true, and the rest of the compilers would check.

Reply:
Done!